sdk/search/azure-search/out/azure/search/index/_credential.pyi

```diff
@@ -0,0 +1,7 @@
+from typing import Any
+
+class SearchApiKeyCredential:
+    def __init__(self, api_key: str) -> None: ...
+    @property
+    def api_key(self): ...
```

Comment:
I assume that you need the key value in your implementation, correct? If not, we should not expose the property...

Reply:
Yes, the client implementation refers to the `api_key` value to insert it in the necessary header.